GWAS/Region_Extraction.ipynb

```diff
@@ -227,14 +228,20 @@
     "        return z\n",
     "\n",
     "    def plink_slice(p,pb=None,pf=None):\n",
+    "        import time\n",
+    "        t = time.localtime()\n",
+    "        \n",
     "        (bim,fam,bed)=p\n",
-    "        if pb is not None:\n",
-    "            bim = bim.iloc[pb]\n",
-    "            bed = bed[pb,:]\n",
-    "        if pf is not None:\n",
-    "            fam = fam.iloc[pf]\n",
-    "            bed = bed[:,pf]\n",
+    "        print(f'{time.strftime(\"%H:%M:%S\", t)}: slicing geno to get bim')\n",
+    "        \n",
+    "        # slice geno to obtain all geno info within specified region\n",
+    "        bim = p[0].loc[(p[0]['chrom'] == str(region[0])) & (p[0]['pos'] > region[1]) & (p[0]['pos'] < region[2])]\n",
```

Comment:
@tfabiha please use `>=` and `<=` to define region boundaries

Reply:
Removed the duplication and redefined region boundaries